CONTRIBUTING.md

```diff
@@ -253,6 +246,19 @@ one business day may be closed (they can always be reopened later).
 If you're still working on something, continue iterating on your branch without
 creating a pull request until it’s ready for review.
 
+### Addressing code review comments
+
+Creating additional commits to address reviewer feedback is generally preferred
+over amending and force-pushing. This makes it easier for reviewers to see what
+has changed since their last review.
+
+Pull requests are always squashed and merged. Before merging, please review and
```

Comment:
nit: start a new line for a sentence.

Reply:
I think this might be a GitHub code review UI issue. I'm not seeing a problem in my editor or in the rich markdown preview.